lib/console/net_ids/net_ids.ex

```diff
@@ -9,18 +11,35 @@ defmodule Console.NetIds do
     Repo.get_by(NetId, [value: id])
   end
 
-  def create_net_id!(attrs \\ %{}, %Organization{} = organization) do
-    attrs = Map.merge(attrs, %{"organization_id" => organization.id})
-
-    %NetId{}
-    |> NetId.changeset(attrs)
-    |> Repo.insert!()
+  def get_net_id!(%User{} = current_user, id) do
+    if current_user.super do
+      Repo.get!(NetId, id)
+    else
+      query = from n in NetId,
+        join: m in Membership, on: m.organization_id == n.organization_id,
+        where: m.user_id == ^current_user.id and n.id == ^id
+      Repo.one!(query)
+    end
   end
 
+  # def create_net_id!(attrs \\ %{}, %Organization{} = organization) do
```

Comment:
should this be commented out?

Reply:
we don't use it since net_ids are currently manually created